src/opencl_sha256crypt_fmt_plug.c

```diff
@@ -573,7 +573,7 @@ static int crypt_all(int *pcount, struct db_salt *_salt)
 		output_hashes = calculated_hash;
 	}
 
-	if (new_keys) {
+	if (ocl_autotune_running || new_keys) {
```

Comment:
As per the PR description this is not necessary in 'my formats' (initialization always happens). Anyway, it doesn't harm.